application/config/routes.rb

```diff
@@ -8,12 +8,20 @@
   get "uploads" => "uploads#index", as: :uploads
   get "uploads/files" => "uploads#files", as: :uploads_files
   post "uploads/:project_id/:file_id/cancel" => "files#cancel", as: :uploads_file_cancel
+  post "uploads/:project_id/:collection_id/add" => "upload_files#add", as: :uploads_file_add
+  get "uploads/:project_id/:collection_id/files" => "upload_files#files", as: :uploads_file_files
+  delete "uploads/:project_id/:collection_id/:file_id" => "upload_files#delete_file", as: :uploads_file_delete
+  post "uploads/:project_id/create" => "upload_files#create_collection", as: :uploads_file_create_collection
 
   get "uploads/create" => "uploads#create", as: :uploads_create #temporally created for tests
 
-  resources :projects, only: [:index, :create, :update, :destroy]
+  resources :projects, only: [:index, :show, :create, :update, :destroy]
```

Comment:
I think only: option can be removed because we are using all the CRUD methods defined in the `resources` convention

Reply:
fixed